### DISCUSSION TITLE
Find if item requires Transparency

### DISCUSSION THREAD

**Discussion by civilCornball** [amzn/selling-partner-api-models#4143] [Q&A]
Say I've had an item in our in-person store for a while.

If I only have its ASIN, how do I tell if it requires a Transparency barcode?

Similar discussion:
https://github.com/amzn/selling-partner-api-models/discussions/3745
(which I don't entirely understand.)

**Comment by jawheral** [comment]
Hi @civilCornball,

From SP-API perspective, an [order item](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#orderitem) that you get from the [getOrderItems API](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#getorderitems) provides a flag `isTransparency` which tells whether the relevant ASIN is enrolled in Transparency.

If you want to learn more about Transparency or enroll products in the Transparency program, please visit this [page](https://brandservices.amazon.com/transparency).

Best,
Jawher
Selling Partner API

**Reply by civilCornball** [reply]
I saw that, earlier, but we have _so many orders_ to sift through.

**Reply by jawheral** [reply]
@civilCornball You can visit the Order Reports [column picker](https://sellercentral.amazon.com/order-reports-and-feeds/column-picker) page on Seller Central. Here you can activate the `isTransparency` column. After this, your order reports should include this data column to check your orders at once.

I hope this helps!

Best,
Jawher
Selling Partner API Solutions Architect

**Reply by civilCornball** [reply]
A little? To add some more information:

We're trying to use the selling partner API to check if a given ASIN in our database has Transparency.

We have >25,000 ASINs in our database, and we've been around for a while.

If we downloaded the order reports, it would take ages while also wasting both our and Amazon's money.

It would be so much easier, and cheaper, if we could simply find if an item has Transparency from a database call like [getCatalogItem](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v0-reference#getcatalogitem).

**Reply by jawheral** [reply]
Hi @civilCornball, 
Data on Product enrolment in Transparency is currently not accessible through SP-API. 
I took your feedback to our product teams and I will share in case I have any updates.

Best,
Jawher
Selling Partner API

**Reply by jugi92** [reply]
Any news on this @jawheral? We are looking for the same information. We were expecting it to be in the [Fullfillment Inbound API](https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v2024-03-20-reference) but its not.

**Reply by thirdpartycook** [reply]
Same request here. The getListingsItem call would be a good place to retrieve this information.
